src/Compilers/Server/VBCSCompiler/IClientConnection.cs

```diff
@@ -14,12 +14,6 @@ namespace Microsoft.CodeAnalysis.CompilerServer
     /// </summary>
     internal interface IClientConnection : IDisposable
     {
-        /// <summary>
-        /// A value which can be used to identify this connection for logging purposes only.  It has 
-        /// no guarantee of uniqueness.  
-        /// </summary>
-        string LoggingIdentifier { get; }
```

Comment:
This field was used to identify client connections for logging purposes in the server. Now that there is a Guid that tracks the connection in both client and server this just isn't needed anymore.